aicsimageio/aics_image.py

```diff
@@ -75,6 +87,22 @@ class AICSImage:
 
     >>> img = AICSImage("malformed_metadata.ome.tiff", reader=readers.TiffReader)
 
+    Data for a mosaic file is returned pre-stitched (if the base reader supports it).
+
+    >>> img = AICSImage("big_mosaic.czi")
+    ... img.dims  # <Dimensions [T: 40, C: 3, Z: 1, Y: 30000, X: 45000]>
+
+    Data for mosaic file can be explicitly returned as tiles.
+    This is the same data just each tile is it's
```

Comment:
each tile is it's what?

Reply:
I think this was supposed to say:

"This is the same data except each tile has it's own dimension index."

Reply:
Resolving with:

"This is the same data as a reconstructed mosaic file except that the tiles are stored in their own dimension (M)."